lib/formatters/getById.js

```diff
@@ -1,23 +1,25 @@
 'use strict';
 
-module.exports = function (rows) {
-  var tags;
+module.exports = function (chal) {
+  var tags = [];
 
   // if no challenge was found
-  if (rows.length === 0) {
-    return {};
+  if (chal.length === 0) {
+    return [];
   }
 
+  if (chal[0].tags_id) {
+    tags = chal.map(function (obj) {
+      return {
+        id: obj.tags_id,
+        name: obj.tags_name
+      };
+    });
+  }
 
-  tags = rows.map(function (obj) {
-    return {
-      id: obj.tags_id,
-      name: obj.tags_name
-    };
-  });
-  delete rows[0].tags_id;
-  delete rows[0].tags_name;
-  rows[0].tags = tags;
+  delete chal[0].tags_id;
```

Comment:
Yeah it is easier to understand as well if we explicitly say which fields you want to keep as opposed to the ones you want to remove. Agree that this is low priority.